frontend/src/views/auth/Login.js

```diff
@@ -1,7 +1,52 @@
-import React from "react";
+import React, { Component } from "react";
 import { Link } from "react-router-dom";
 
-export default function Login() {
+class App extends Component {
+    state = {
+      response: '',
+      post: '',
+      responseToPost: '',
+      email:'',
+      password:'',
+      role:'',
+    };
+    
+    componentDidMount() {
+      this.callApi()
```

Comment:
Why is the component trying to make a login request without the user providing all the necessary details 🤔

Reply:
That was for a previous implementation that is now scrapped. I have now used the latest structure as agreed upon earlier by the team.